plugins/extra.py

```diff
@@ -83,3 +86,35 @@ async def _(e):
         )
     else:
         await e.try_delete()
+
+
+@ultroid_cmd(
+    pattern="delmsgs",
+)
+async def delete_messages(event):
+    # Get the search phrase from the command
+    search_phrase = event.raw_text.split(" ", 1)[1]
+
+    # Get the chat ID of the group
+    chat_id = event.chat_id
+
+    # Get the messages in the chat
+    async for message in nimbus_bot.iter_messages(chat_id):
```

Comment:
`nimbus_bot` ?

Reply:
Ahh right thanks for that this is what I said before as I've tested and it all works.. I'll fix this